mne/viz/evoked.py

```diff
@@ -1623,7 +1623,12 @@ def whitened_gfp(x, rank=None):
     _validate_type(axes, (list, tuple, np.ndarray, None), "axes")
     if axes is None:
         _, axes = plt.subplots(
-            n_rows, n_columns, sharex=True, sharey=False, figsize=(8.8, 2.2 * n_rows)
+            n_rows,
+            n_columns,
+            sharex=True,
+            sharey=False,
+            figsize=(8.8, 2.2 * n_rows),
+            constrained_layout=True,
```

Comment:
Third code change: use a contrained_layout to fix this kind of plot:

![Screenshot from 2023-09-28 12-51-19](https://github.com/mne-tools/mne-python/assets/73893616/f2c270d5-48c9-4635-9e12-b9b7e8e6e022)